.github/workflows/pkgbuild.yml

```diff
@@ -0,0 +1,56 @@
+#
+# Verify on CI/GHA that package building works.
+#
+# TODO: Currently, this supports RPM-based systems only.
+#       By adjusting a few details, it can be made work
+#       for other distribution types as well.
+#
+name: Test packaging
+
+on:
+
+  # On which repository actions to trigger the build.
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allow job to be triggered manually.
+  workflow_dispatch:
+
+# Cancel in-progress jobs when pushing to the same branch.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+
+  tests:
+
+    runs-on: "ubuntu-latest"
+    strategy:
+
+      # Run all jobs to completion (false), or cancel
+      # all jobs once the first one fails (true).
+      fail-fast: false
+
+      # Define a list of build targets. The labels should match the
+      # items within the `services` section of `docker-compose.yml`.
+      matrix:
+        target: [
+          "rpmbuild.el8",
+        ]
+
+    name: Target ${{ matrix.target }}
+    steps:
+
+    - name: Acquire sources
+      uses: actions/checkout@v3
+
+    - name: Build package using Docker
+      run: |
+        docker compose run --user root --rm ${{ matrix.target }} build-rpm.sh
+
+    - name: Verify package has been produced
+      run: |
+        ls dist/rpm/*.rpm
```

Comment:
Since it's possible for the `src.rpm` to be build here, but not the actual `apprise` package.  is it as simple as changing this to:
```bash
ls dist/rpms/*.noarch.rpm
```

That should validate that the RPMs required are present.  Pretty sure the `build-rpm.sh` script will have a non-zero exit if any of the build process fails; but if not, i'll make it so that's the case.  So the likelihood is we'll catch every problem on that step anyway :slightly_smiling_face:

Reply:
The output on my machine is:
```
$ ls dist/rpm/*.rpm
dist/rpm/python-apprise-1.2.0-1.el8.src.rpm
```

There is no `noarch` in there.

Reply:
OMG. I missed it. Thank you!

Reply:
That would be correct, right?

```
$ ls -alF dist/rpm/noarch/*.noarch.rpm
-rw-r--r--  1 amo  staff    20824 Dec 10 18:05 dist/rpm/noarch/apprise-1.2.0-1.el8.noarch.rpm
-rw-r--r--  1 amo  staff  1072960 Dec 10 18:05 dist/rpm/noarch/python3-apprise-1.2.0-1.el8.noarch.rpm
```

Reply:
That's perfect! the 2 files you see there are what we expect

Reply:
I've edited the patch. It looks like it's working as intended now.

-- https://github.com/caronc/apprise/actions/runs/3665277903/jobs/6196242239#step:4:1